skopt/space/transformers.py

```diff
@@ -108,15 +108,15 @@ class Normalize(Transformer):
         to `True` when applying this transform to integers.
     """
     def __init__(self, low, high, is_int=False):
-        self.low = low
-        self.high = high
+        self.low = float(low)
+        self.high = float(high)
         self.is_int = is_int
 
     def transform(self, X):
         X = np.asarray(X)
-        if np.any(X > self.high):
+        if np.any(X > self.high + 1e-8):
```

Comment:
Unsure if this is really the best way of doing this. Sometimes `scipy.optimize` passes in a value that is just ever so slightly larger than the boundary, so this stops the exception from being raised.

Reply:
Yes I actually remember having some boundary checking issues some time ago, so I think this is a good idea.

Reply:
Is there a better way to do this? `np.allclose` uses a combination of absolute and relative tolerances to handle large numbers as well. Or should we go with this for now and improve later?

Reply:
Both are fine with me tbh. It seems like `np.allclose` might be a bit more advanced for use int these situations.